### ISSUE TITLE
Port "About" page into a blog post and remove as top level page

### ISSUE THREAD

**Issue by daiyi** [daiyitastic/mypaint.github.io#3]
The current About page has really interesting history in it, but I would expect an organisation/project's about page to talk about features or the team or convince me why I should use the tool, all of which we've brought to the landing page for visibility. The top navigation is prime real estate and should be trim as possible. I suggest moving forward:

1. port the contents of the current "about" page into a blog post to preserve the interesting historical facts therein
2. remove `/about` because the information in it would be redundant to the landing page

I strongly believe this is the right way to go, unless there is particular sentimental reason for this history to be super visible. Let me know if there's problems with this! cc: @odysseywestra

**Comment by odysseywestra** [comment]
I agree. It was just something just that was put together to help fill some of the gaps of the website. You can move to a new created folder and call it [_drafts so Jekyll won't generate a page or blog post](https://jekyllrb.com/docs/drafts/).

**Comment by odysseywestra** [comment]
That also allows us to re-introduce the page as history of MyPaint or whatever.
